File: grader.py
import sys

# Defining the Grader class
class Grader:
	# Creating the constructor
	def __init__(self):
		# Getting the user data from the command line in the constructor
		self.name = raw_input("What is your name? ")
		self.assignment_name = raw_input("What is the name of your assignment? ")
		self.grade = float(raw_input("What was your grade in " + self.assignment_name + "? "))
	
	# Determine the users letter grade based on the grade they entered
	def getGrade(self):
		letter_grade = ''
		
		if self.grade < 60:
			letter_grade = 'F'
		elif self.grade < 70:
			letter_grade = 'D'
		elif self.grade < 80:
			letter_grade = 'C'
		elif self.grade < 90:
			letter_grade = 'B'
		else:
			letter_grade = 'A'
		
		return letter_grade

# Instantiate the Grader class
myGrade = Grader()
# Display the letter grade for the user
print("Your letter grade in " + myGrade.assignment_name + " was a(n) " + myGrade.getGrade())
		

File: grader.js
const readline = require("readline")
const rl = readline.createInterface({
	input : process.stdin,
	output : process.stdout
})

class Grader{
	constructor(){
	
	}
	
	getGrade(){
	
		rl.question("What is your name? ", (name) => {
		
			rl.question("What is the name of your assignment? ", (assignmentName) => {
			
				rl.question("What is your grade in " + assignmentName + "? ", (grade) => {
				
					let letterGrade
					
					if(grade < 60){
						letterGrade = 'F'
					}else if(grade < 70){
						letterGrade = 'D'
					}else if(grade < 80){
						letterGrade = 'C'
					}else if(grade < 90){
						letterGrade = 'B'
					}else{
						letterGrade = 'A'
					}
					
					console.log(letterGrade)
				
				})
			})
		})
	}
}

const myGrade = new Grader()
myGrade.getGrade()